tfs/hdf.py

```diff
@@ -0,0 +1,93 @@
+"""
+HDF5 Tools
```

Comment:
I would rather name this `HDF5 I/O` so the section in the documentation is extra clear.